doc/SONIC_Test_Ingress_Discard_All_HLD.md

```diff
@@ -0,0 +1,738 @@
+### [DRAFT, UNDER DEVELOPMENT]
+
+- [Overview](#overview)
+    - [Scope](#scope)
+    - [Supported topologies](#supported-topologies)
+    - [Discard groups covered by test case](#discard-groups-covered-by-test-cases)
+    - [Related DUT CLI commands](#related-dut-cli-commands)
+    - [SAI attributes](#sai-attributes)
+- [General test flow](#general-test-flow)
+- [Run test](#run-test)
+- [Test cases](#test-cases)
+    - [Test case #1](#test-case-1)
+    - [Test case #2](#test-case-2)
+    - [Test case #3](#test-case-3)
+    - [Test case #4](#test-case-4)
+    - [Test case #5](#test-case-5)
+    - [Test case #6](#test-case-6)
+    - [Test case #7](#test-case-7)
+    - [Test case #8](#test-case-8)
+    - [Test case #9](#test-case-9)
+    - [Test case #10](#test-case-10)
+    - [Test case #11](#test-case-11)
+    - [Test case #12](#test-case-12)
+    - [Test case #13](#test-case-13)
+    - [Test case #14](#test-case-14)
+    - [Test case #15](#test-case-15)
+    - [Test case #16](#test-case-16)
+    - [Test case #17](#test-case-17)
+    - [Test case #18](#test-case-18)
+
+#### Overview
+The purpose is to test "RX_DRP" counter got from show command "show interfaces counters" triggers on receiving specific packets by DUT.
+The "RX_DRP" counter counts all discard events. This counter counts concurrently with other discard counters.
+The test assumes all necessary configuration are already pre-configured on the SONIC switch before test runs.
```

Comment:
Could you provide a short description of what this configuration is?

Reply:
Looks like this sentence is confusing. Tests do not require some specific configurations.
As I understand discards which are going to be tested are enabled by default. Only ACL will be configured by test cases itself.

The only important thing is to disable control traffic before test run.
So I will update this sentence.